app/models/host_genome.rb

```diff
@@ -2,14 +2,39 @@ class HostGenome < ApplicationRecord
   has_many :samples, dependent: :restrict_with_exception
   has_and_belongs_to_many :metadata_fields
 
+  before_create :add_default_metadata_fields
+
+  validates :name, presence: true, uniqueness: { case_sensitive: false }, format: {
+    with: /\A[A-Z][\w|\s|\.|\-]+\z/,
+    message: "allows only word, period, dash or space chars, and first char must be capitalized: %<value>",
+  }
+
+  validates :s3_bowtie2_index_path, :s3_star_index_path, format: {
+    with: %r{\As3://.*\z},
+    message: "should be located in s3.",
+  }
+
+  # NOTE: not sure why this columns was not created as boolean
+  validates :skip_deutero_filter, inclusion: { in: [0, 1] }
+
   validates :taxa_category, inclusion: { allow_blank: true, in: [
     # From "Sample Type Groupings" at
     # https://docs.google.com/spreadsheets/d/1_hPkQe5LI0Zw_C0Ls4HVCEDsc_FNNVOaU_aAfoaiZRE/
     "human",
     "insect",
   ], }, if: -> { respond_to?(:taxa_category) } # for migrations
 
-  before_create :add_default_metadata_fields
+  ERCC_PATH_PREFIX = "s3://idseq-database/host_filter/ercc/2017-09-01-utc-1504224000-unixtime__2017-09-01-utc-1504224000-unixtime/".freeze
```

Comment:
can this path come from the "ERCC Only" host genome instead potentially? I could see someone adding a "2020 ERCC" update and forgetting to change this.

Reply:
Good idea.

My question then is: if we upgrade to ERCC 2020, would we update the existing ERCC host genome or create a new one? I guess it depends on the nature of ERCCs.

Reply:
In the past, we have updated host genomes like the mosquito in-place, but I guess it depends